src/fides/api/ctl/routes/datamap.py

```diff
@@ -163,8 +163,7 @@ async def export_datamap(
 
     formatted_datamap = format_datamap_values(joined_system_dataset_df, custom_columns)
 
-    # prepend column names
-    formatted_datamap = [DATAMAP_COLUMNS_API] + formatted_datamap
+    formatted_datamap = [{**DATAMAP_COLUMNS_API, **custom_columns}] + formatted_datamap
```

Comment:
The main reason they were not populating is the custom field was not populated into the first item in the response list, which the UI uses to render columns. This fixes that by including the custom fields that exist